constraints.txt

```diff
@@ -1,5 +1,6 @@
 cftime<1.1.1;python_version=='3.5'
+cftime<1.1.1;python_version=='3.6'
 pandas<0.25.0;python_version=='3.5'
 xarray<0.14.0;python_version=='3.5'
 netCDF4<1.5.4;python_version=='3.5'
```

Comment:
It might be safest to also duplicate this constraint (netCDF4) for python 3.6, as that is the package that will call cftime.